lib/dom.js

```diff
@@ -410,6 +411,7 @@ declare class Element extends Node {
     removeAttribute(name?: string): void;
     removeAttributeNS(namespaceURI: string, localName: string): void;
     removeAttributeNode(oldAttr: Attr): Attr;
+    requestFullscreen(): void;
```

Comment:
`requestFullscreen()` isn't a standard DOM method. It's experimental, but I don't think that's enough to add to the core library definition.